modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1023,24 +1040,46 @@ def _read(cls, filepath_or_buffer, **kwargs):
             )
 
         is_quoting = kwargs["quoting"] != QUOTE_NONE
+        usecols = kwargs["usecols"]
```

Comment:
```suggestion
        usecols = kwargs.get("usecols")
```
so it would not raise a keyerror if the key is not in the dict

Reply:
Sure, but at this level, all options are in the dictionary. It makes no sense to use default values in several places. This makes the code difficult to maintain.
The main place where default values are defined should be the function signature.